templates/category-page/partials/ckan_result_template.php

```diff
@@ -98,7 +98,7 @@
       $source = wp_odm_solr_parse_multilingual_ckan_content($document->extras_odm_source,odm_language_manager()->get_current_language(),"");
       if (!empty($source)):
         if (strlen($source) > 40):
-          $source = substr($source,0,40) . " ...";
```

Comment:
There is no space between strings and 3 dots in English? I saw that in Khmer news, the editors add the space before dots. 
![screenshot_1](https://user-images.githubusercontent.com/6613530/28808386-cce3244e-76a5-11e7-9eda-944bb5701d39.jpg)